ekf/tests/test_runner/result_check.c

```diff
@@ -0,0 +1,307 @@
+/*
+ * Phoenix-Pilot
+ *
+ * Part of EKF test runner
```

Comment:
If we navigated here we know its part of EKF test runner :)
```suggestion
 * EKF test runner result checking routines
```
Please amend also the header file.